### ISSUE TITLE
NO DETECTIONS

### ISSUE THREAD

**Issue by mlandi96** [ultralytics/yolov5#10788]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

i've trained the yolov5 with my custum dataset and i haven't got any predictions

what should i do please ? have i missed something ? 

I've used a pre-trained model with a small dataset of 81 images, for just one class to detect

`python segment/train.py --img 640 --batch 16 --epochs 10 --data my_data_config.yaml --weights weights/yolov5s-seg.pt`

thanks
![Capture d’écran du 2023-01-18 08-57-18](https://user-images.githubusercontent.com/115972997/213096283-9bde123b-b62f-46bf-ac6a-762255d19089.png)

**Comment by glenn-jocher** [comment]
@mlandi96 hi there,

It looks like you trained YOLOv5 with a custom dataset of 81 images for a single class, which is great! 🚀 

To diagnose the issue, try running inference on a sample test image using the newly trained model, ensuring proper path references, image formatting, and matching class labels. It's also beneficial to check that the model has converged during training, reflected in decreasing loss values.

For further troubleshooting, please refer to our documentation at https://docs.ultralytics.com/yolov5/ for additional guidance on dataset preparation, training, and inference.

Let us know if the issue persists, and we'll be happy to assist further.

Thanks!
